README.md

```diff
@@ -26,10 +26,8 @@ they may be customized as needed.
 | AZURE_APIM_HOST             | The host name of the API Management service                                                      | string |
 | AZURE_APIM_RESOURCE_GROUP   | The name of the resource group used to get the subscriptions                                     | string |
 | AZURE_SUBSCRIPTION_ID       | Credentials which identify the Azure subscription, used to init the APIM  client                 | string |
-| UserDataArchiveStorageConnection | Storage connection string to store zip file for user to download their data | string |
```

Comment:
@gunzip it's still used here: https://github.com/pagopa/io-functions-admin/blob/e50ba7fe3ef4c7c47dc39471afafb14eb3c35ad2/ExtractUserDataActivity/index.ts#L78

We should change it as well. Should we use `StorageConnection`? I guess we meant to keep user data bundles separated from normal IO's working storage

Reply:
no sorry it's my fault, I'll revert it into the readme again :)